### ISSUE TITLE
Encoding Conformance Classes

### ISSUE THREAD

**Issue by dstenger** [opengeospatial/ets-gpkg12#26]
See http://opengeospatial.github.io/teamengine/encoding-conformance-classes.html

All basic conformance classes have to be identified and added to config.xml.

**Comment by jyutzler** [comment]
@dstenger can you provide an example of where this is done properly? I don't understand what you are asking me to do here.

**Comment by keshavnangare** [comment]
@jyutzler 

To identify the conformance classes in HTML report we need to add it in config,xml file of test-suite.

You can refer documentation link provided in first comment.

For example: ets-cat30 --> https://github.com/opengeospatial/ets-cat30/blob/master/src/main/config/teamengine/config.xml

see the element `"<BasicConformanceClasses>"`

**Comment by dstenger** [comment]
@jyutzler Thank you for the fix.

Did you put following line into the commit on purpose?
```
<entry key="ics">Core,Tiles,Features,Attributes,Extensions,Non-linear Geometries,RTree Index,Webp,Metadata,Schema,CRS WKT,Tiled Gridded Elevation Data</entry>
```

**Comment by jyutzler** [comment]
@dstenger Actually, no, that is an error. Fixed by #29.
